### PR TITLE
Further improvements of vSphere documentation with PR comments

### PR DESCRIPTION
**Pull request by embik**
This is a follow-up to #1448, which got merged by accident, addressing the comments from @hc2p.

### CODE REVIEW

File: content/kubermatic/main/architecture/supported-providers/vsphere/vsphere.en.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ weight = 7
 
 +++
 
+KKP fully supports VMware vSphere as a tier 1 provider. It allows the automatic provision of user cluster nodes and automatically deploys CCM and CSI for cloud provider and storage integration.
```
Comment (hc2p) [inline content/kubermatic/main/architecture/supported-providers/vsphere/vsphere.en.md:8]:
```suggestion
KKP fully supports VMware vSphere as a tier 1 provider. It allows the automatic provision of user cluster nodes and management of storage through the deployment and integration of vSphere CCM and CSI.
```

Comment (embik) [reply content/kubermatic/main/architecture/supported-providers/vsphere/vsphere.en.md]: done, updated.

**Comment by embik** [comment]
@hc2p can you please take another look?

**Review by hc2p — APPROVED** [review]
lgtm

**Pull request merged**